modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,139 @@
+
+== Game effects of {portal}
+
+Where your army includes any units or objectives that have the {portal} ability:
+
+* Regardless of the scenario, you may keep any detachments that contain units that can use a portal out of the play area, as <<Portal Reserves>>.
+
+Where your unit or objective has the {portal} ability:
+
+* You can use this unit to move portal reserves detachments into play after they arrive.
+You can also move detachments that are in play between one portal and another.
+See <<portal-moves>>.
+
+[[portal-moves]]
+== Use a {portal} unit to move your forces
+
+You can use portals to move some of your forces around.
+Where you wish to move one of your detachments into or out of a portal, and it contains permitted units:
+
+* You may do this only in the Movement phase.
+// TODO: Is this stipulation necessary? On the one hand we're trying to apply standard transport rules for simplicity, but on the other hand this kind of stipulation sets up differences from those rules which complicates things again and creates logical gaps.
+* You may do this only if you give the detachment Advance or Assault orders.
+* Use the normal xref:main-rules:transportation.adoc[] rules to embark and disembark the detachment's units.
+footnote:[
+When you disembark units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
+In this case, just deduct 5 cm from the movement of each unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
+]
+// TODO: Do we need to mention something to the effect that portal reserves detachments either (a) already count as embarked so don't pay 5 cm penalty to embark, or (b) need to pay the penalty to embark, just as units already in play need to do? Also, given that for a detachment that is already in play you must embark and then immediately disembark from another portal, won't this get a bit complicated given that you must also pay a 5 cm penalty for *each* portal unit...? 
+* You may use each individual portal only *once per turn*, to embark or disembark a single detachment.
+Any portal that you use becomes unavailable until the start of the next turn.
+* You may move a detachment that is already in the play area into an available portal -- but only if you immediately disembark the detachment from a different available portal elsewhere in the play area.
+You cannot use portals to transfer detachments *out* of the play area during the game.
+
+== Portal Reserves
+
+These Portal Reserves rules are a special variation on the standard Reserves rules.
+The difference is in the detail of how detachments that you set aside as portal reserves will arrive, and how you put them into play.
+
+IMPORTANT: You can put portal reserves detachments into play *only* by means of {portal} units.
```

Comment:
Following on from the earlier comment, I think this needs to mention 'disembarking' in some way.

Maybe this?

```suggestion
IMPORTANT: You can put portal reserves detachments into play *only* by disembarking from {portal} units.
```

Reply:
I **deliberately** avoided this terminology at this point.

I framed the note in more general terms because we haven't yet made it plain that we will apply the same specific terminology (and much the same mechanisms) as we used in the standard transportation rules. That becomes clear just a little later, after this note (notwithstanding any further work we need to do to improve the draft).

It's not critical, most people will get it without being thrown off, but on the whole I'd argue that it's better to stick to the more general phrasing for the more general point that comes relatively early, and only switch to the specific phrasing once we've made it plain in this context and we're talking about the specifics.

Reply:
I'll leave it to you, as you have final say and we have different approaches to this kind of thing.

Personally I try to only use general phrasing if I need to cover multiple options. If there is only one option you're swapping phrasing partway through the text, for no benefit in readability.

Reply:
> I'll leave it to you, as you have final say and we have different approaches to this kind of thing.

👍I appreciate that. But just so you know, I don't think our approaches are so different here:

> Personally I try to only use general phrasing if I need to cover multiple options. If there is only one option you're swapping phrasing partway through the text, for no benefit in readability.

I agree with this in general. But in this particular case, at this particular point in the text, your suggestion would mean that we'd use a term that has specific connotations before we've actually introduced the term with its connotations for this context (that happens a few paragraphs later). The way I've approached it is **progressive disclosure**, not 'swapping phrases'.

No biggie, just thought I'd elucidate my rationale in case it helps with similar cases later. You're certainly right to keep an eye out for needless changes in how we phrase particular rules concepts, similar to how we avoid synonyms for significant terms.

Reply:
Actually, I'm talking half sense and half out of my arse — progressive disclosure is valid, but the text does introduce embark/disembark terms higher up the page... This topic and the shuffling around made my brain hurt, and I lost track a bit! 😂

Anyway, I don't think it matters much — we aren't using needless synonyms, which is definitely bad, and I don't think that general phrases like 'put into play by means of' don't really detract from or confuse more specific use of the terms that define those 'means'.

It might change some more anyway by the time we complete this round of editing. 😅 (Although, we're on the home straight now I think, and worst case we could live with a few open questions on this experimental topic.)